### PR TITLE
+doc #20046 allow general sections to be searched

### LINKED ISSUE #20046: Algolia search enable `general` pages
We should include this filter:

```
facetFilters: [
  ["language:scala","language:general"],
  "version:VERSION"
]
```

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #20046

### CODE REVIEW

File: akka-docs/_sphinx/themes/akka/layout.html [hunk 1]
```diff
@@ -217,10 +217,13 @@
...
     apiKey: '543bad5ad786495d9ccd445ed34ed082',
     indexName: 'akka_io',
-    inputSelector: '#q',
+    inputSelector: '#search',
     algoliaOptions: {
       hitsPerPage: 10,
-      facetFilters: '["language:' + lang + '","version:' + version + '"]'
+      facetFilters: '[' +
+      '"language:' + lang + '"' +
+      '"language:general"' +
```
Comment (johanandren) [inline akka-docs/_sphinx/themes/akka/layout.html]: this is the actual fix?

Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/layout.html]: dang, missed a `,`, but yes. Sorry, I did not CC entire team on algolia mailings on this one – they've added additional indexing now basically.

```
I've added the "general" pages. It's in the facet "language". So to search on Scala or General page, and a specific VERSION just write: 
facetFilters: [
  ["language:scala","language:general"],
  "version:VERSION"
]
```

Comment (johanandren) [inline akka-docs/_sphinx/themes/akka/layout.html]: Ok, nice!

**Comment by johanandren** [comment]
LGTM when missing comma has been added :)

File: akka-docs/_sphinx/themes/akka/layout.html (hunk 1 above)
Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/layout.html]: Hmm, need to do some more local testing

Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/layout.html]: Everything is wrong. Reworking query...

**Comment by ktoso** [comment]
Fixed and verified.

**Pull request merged**